build.py

```diff
@@ -254,7 +253,8 @@ def core_cmake_args(components, backends, install_dir):
             else:
                 fail('unknown core backend {}'.format(be))
 
-    cargs.append('-DTRITON_EXTRA_LIB_PATHS=/opt/tritonserver/lib')
```

Comment:
I don't think we need TRITON_EXTRA_LIB_PATHS anymore. Can you try removing it. We should also not need --opt-dir flag.